docs/examples.rst

```diff
@@ -471,3 +471,38 @@ like so:
 .. include::  ../tests/core/contracts/test_contract_example.py
     :code: python
     :start-line: 1
+    
+Using Infura Rinkeby Node
+=========================
+Import your required libraries
+
+.. code-block:: python
+
+    from web3 import Web3, HTTPProvider
+    
+Intialize your infura node in a variable
+
+.. code-block:: python
+    
+    w3 = Web3(Web3.HTTPProvider("https://rinkeby.infura.io/v3/YOUR_INFURA_KEY"))
+
+ 
+Inject the middle-ware in the variable holding your infura node;
```

Comment:
yep.

actually, i'm bit a beginner at this, so i thought to be a bit more 'explicit', so that other beginners find this a bit more approachable and clear